src/server/actions/home-queries.yaml

```diff
@@ -0,0 +1,5 @@
+retrieveVenues: >
+    SELECT a.*, v.id venue_id, v.name venue_name, ps.id space_id, ps.name space_name
+    FROM performance_space ps, venue v, address a
+    WHERE ps.venue_id = v.id
+    AND   v.address_id = a.id
```

Comment:
I prefer explicit joins:

    SELECT *, etc FROM performance_space ps
    INNER JOIN venue v ON ps.venue_id = v.id
    LEFT JOIN address ON v.address_id = a.id

Makes it easier to distinguish between join logic and filtering.